pandas/core/series.py

```diff
@@ -185,6 +185,8 @@ class Series(base.IndexOpsMixin, generic.NDFrame):
         base.IndexOpsMixin.hasnans.func, doc=base.IndexOpsMixin.hasnans.__doc__
     )
     _data: SingleBlockManager
+    div: Callable[["Series"], "Series"]
+    rdiv: Callable[["Series"], "Series"]
```

Comment:
I assume the first argument to the Callable would be self in this case. should we have other typed?

Reply:
other can be just about anything

Reply:
@simonjayhawkins @jbrockmendel 
Does that mean the annotation should be `Callable[["Series", Any], "Series"]`?